docs/reference/items.md

```diff
@@ -1053,6 +1155,95 @@ Empty body.
 }
 ```
 
+</template>
+<template #graphql>
+
+`POST /graphql`
+
+```graphql
+mutation {
+	delete_articles_items(ids: [15, 16, 21]) {
+		ids
+	}
+}
+```
+
+</template>
+<template #sdk>
+
+```js
+import { createDirectus, rest, deleteItems } from '@directus/sdk';
+
+const client = createDirectus('https://directus.example.com').with(rest());
+
+const result = await client.request(deleteItems('articles', ['6', '7']));
+```
+
+</template>
+</SnippetToggler>
+
+## Delete Items by Query
+
+Delete multiple existing items using a query object.
+
+<SnippetToggler :choices="['REST', 'GraphQL', 'SDK']" group="api">
+<template #rest>
+
+`DELETE /items/:collection`
+
+Provide an object containing `query` as your request body.
+
+</template>
+<template #graphql>
+
+`POST /graphql`
+
+```graphql
+type Mutation {
+	delete_<collection>_batch(
+		filter: <collection>_filter,
+		sort: [String],
+		limit: Int,
+		offset: Int,
+		page: Int,
+		search: String,
+	): delete_many
+}
+```
```

Comment:
The `delete_<collection>_batch` mutation does not exist as `delete_<collection>_items` should suffice. The `delete_<collection>_by_query` would be implemented separately such as through https://github.com/directus/directus/pull/24101.

Reply:
I don't think `_items` suffices, as it takes an array of IDs rather than a query in order to select what items to delete 👎🏻